src/python/pants/backend/jvm/tasks/jvm_compile/rsc/rsc_compile.py

```diff
@@ -133,6 +133,9 @@ def mirrored_target_option_actions(self):
   def implementation_version(cls):
     return super().implementation_version() + [('RscCompile', 173)]
 
+  def additional_confs_for_fignerprinting(self):
```

Comment:
Hmm... this is _supposed_ to be accounted for by https://github.com/pantsbuild/pants/blob/aa47d931adb9356475f4634821dae91a09aedb9a/src/python/pants/task/task.py#L190-L192 ... is there a missing `Subsystem` dependency here?

Reply:
It doesn't appear that `workflow` was added to the `scala_library()`'s payload, which is what I thought was used for fingerprinting: https://github.com/pantsbuild/pants/blob/aa47d931adb9356475f4634821dae91a09aedb9a/src/python/pants/backend/jvm/targets/scala_library.py#L55-L58.

Reply:
Or in `JvmTarget`: https://github.com/pantsbuild/pants/blob/aa47d931adb9356475f4634821dae91a09aedb9a/src/python/pants/backend/jvm/targets/jvm_target.py#L93-L108

Reply:
@stuhood that makes sense, although in practice changing `--workflow` resulted the same fingerprint, so likely something is missing.

Reply:
@wisechengyi : Perhaps we're not fingerprinting the "enum" type in a useful way...?

Reply:
Tried converting `--workflow` to a str type option, and target fingerprint isn't changed either, so likely it's just not taken into consideration.

Reply:
Possibly false alarm, as I only printing the target key instead of the full key.
https://github.com/pantsbuild/pants/blob/14f079f03bc0fa792241d80944e890c844efedce/src/python/pants/invalidation/build_invalidator.py#L95-L106